### PR TITLE
Add http session API sites view

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

**Comment by psiinon** [comment]
I had problems working out what format the site name was, so this call shows you all of the valid ones :)

**Comment by thc202** [comment]
It should support any HTTP/S URL (it's extracted the authority). Do you recall which site names were failing?

**Comment by psiinon** [comment]
They werent failing, I just had to try a load of different variations of the site name before I hit the one that worked. And I thought that if I dont know what its supposed to look like how would anyone else?

**Comment by thc202** [comment]
OK, could you provide the site names that didn't work? Just want to check if the code supposed to create the expected name is not working correctly (as it seems).

**Comment by psiinon** [comment]
I think I started off trying names like https://www.example.com which is how the site is refered to in other places in the api (I think) when in fact it needed to be like www.example.com:443 (from memory)

**Comment by thc202** [comment]
Odd, that name should have worked, do you recall with which action/view was that happening? (AFAICT all should work fine with the URL you provided.)

Pull request looks good to me.

File: src/org/zaproxy/zap/extension/httpsessions/HttpSessionsAPI.java [hunk 1]
```diff
@@ -231,6 +235,13 @@ public ApiResponse handleApiView(String name, JSONObject params) throws ApiExcep
 
 		HttpSessionsSite site;
 		switch (name) {
+		case VIEW_SITES:
+			// Get existing sessions for all sites
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/httpsessions/HttpSessionsAPI.java]: Thus comment is a bit misleading, sites are returned not sessions.

Comment (psiinon) [inline src/org/zaproxy/zap/extension/httpsessions/HttpSessionsAPI.java]: Doh! Will fix..

**Comment by psiinon** [comment]
Fixed comment

**Comment by kingthorin** [comment]
Looks good

**Pull request merged**